lib/xuclient/XUClient.ts

```diff
@@ -14,91 +14,95 @@ class XUClient {
     if (host) {
       this.host = host;
     }
+    this.port = port || 8886;
+    const xurpc: any = grpc.load(__dirname + '/xud.proto', 'proto');
```

Comment:
this causes the client to fail because you don't copy its `xud.proto` to the `dist` folder

Reply:
you duplicated the protobuf in`xuclient` folder, which makes sense to some degree, because the client actually should be detached from the project.
So while having the file there, you just need to copy to `dist` during the compile procedure.

If you want to use the `lib/grpc/xud.proto` one, then remove the duplication from `xuclient`, and let me know if you need help with targeting the file path.